PF/Assignment1/ConcordanceOfCharacters.java

```diff
@@ -0,0 +1,59 @@
+//Program to find occurrence of elements in a string
+import java.util.HashSet;
+import java.util.Iterator;
+import java.util.Scanner;
+import java.util.Set;
+
+public class ConcordanceOfCharacters {
+
+	int k = 0;
+	Set<Character> setA = new HashSet<Character>();
+	String input;
+	int length;
+	String storeIndex;
+	String result[];
```

Comment:
ACTION: Instead of using different collection to store concordance result and unique characters , you should use HasMap<Character, String>() as collection.